python/lsst/pipe/tasks/dcrAssembleCoadd.py

```diff
@@ -554,10 +569,25 @@ def newModelFromResidual(self, dcrModels, residualGeneratorList, bbox, statsFlag
             # `MaskedImage`s only support in-place addition, so rename for readability
             residual += model[bbox]
             newModel = residual
-            dcrModels.clampModel(subfilter, newModel, bbox, statsCtrl, self.config.regularizeSigma,
-                                 self.config.modelClampFactor, self.config.convergenceMaskPlanes)
-            dcrModels.conditionDcrModel(subfilter, newModel, bbox, gain=1.)
+            # Catch any invalid values
+            badPixels = ~np.isfinite(newModel.image.array)
+            newModel.image.array[badPixels] = model[bbox].image.array[badPixels]
```

Comment:
Is it better to replace these values, or carry around a mask and mask them out of further calculations